features/step_definitions/user_profile_form_steps.rb

```diff
@@ -42,7 +42,7 @@
 
 When(/^there is no (.+) associated with my account$/) do |attribute|
   attribute = attribute.parameterize('_')
-  @user.update_attribute(attribute, nil)
+  @user.update(attribute.to_sym => '')
```

Comment:
Can we just use `@user.update_attribute(attribute, '')`. I don't think that method needs a sym coercion